.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "i18n"]
 	path = i18n
-	url = https://github.com/ourjapanlife/findadoc-localization
+	url = https://github.com/2moon/findadoc-localization.git
```

Comment:
Also, the submodule changes should be removed. 

If you're having trouble doing that, I recommend just making another branch and cherry pick the changes into the new branch. 😄

Reply:
> Once the localization change is merged, I will point the localization submodule back to the main branch

yeah, that's what I was going to do - I'm going to do it now that the other one is merged ~

Reply:
done